logstash/values.yaml

```diff
@@ -0,0 +1,197 @@
+---
+replicas: 1
+
+# Allows you to add any config files in /usr/share/logstash/config/
+# such as logstash.yml and log4j2.properties
+logstashConfig: {}
+#  logstash.yml: |
+#    key:
+#      nestedkey: value
+#  log4j2.properties: |
+#    key = value
+
+# Allows you to add any pipeline files in /usr/share/logstash/pipeline/
+logstashPipeline: {}
+#  uptime.conf: |
+#    input {
+#      exec {
+#        command => "uptime"
+#        interval => 30
+#      }
+#    }
+#    output { stdout { } }
+
+# Extra environment variables to append to this nodeGroup
+# This will be appended to the current 'env:' key. You can use any of the kubernetes env
+# syntax here
+extraEnvs: []
+#  - name: MY_ENVIRONMENT_VAR
+#    value: the_value_goes_here
+
+# A list of secrets and their paths to mount inside the pod
+secretMounts: []
+
```

Comment:
I think leaving in the commented example like the Elasticsearch chart would make it easier for users to know the formatting.

Reply:
Yeah for this one I'd like to add a commented example. Do you have any good use case needing a secret mount in Logstash so we can have a relevant example.

Reply:
Best example would be for mounting certificates to be able to talk to Elasticsearch with security enabled.